src/utils/resources/shared.ts

```diff
@@ -45,6 +45,17 @@ export const getAnnotation = (
 export const getLabel = (entity: K8sResourceCommon, label: string, defaultValue?: string): string =>
   entity?.metadata?.labels?.[label] ?? defaultValue;
 
+/**
+ * function for getting an entity's labels
+ * @param {K8sResourceCommon} entity - entity to get labels from
+ * @param {string} defaultValue - default value to return if no labels are found
+ * @returns the entity's labels or the defaultValue no labels are present
+ */
+export const getLabels = (
```

Comment:
We already have getLabel, and I see you use it in dataSourceHasDynamicSSHLabel(). Is this new function in use?

Reply:
I started off with `getLabels` and then realized using `getLabel` would work better, but for got to remove `getLabels` after I changed it. I just removed `getLabels`.